java/src/main/native/src/ColumnViewJni.cpp

```diff
@@ -2457,4 +2457,46 @@ JNIEXPORT jlong JNICALL Java_ai_rapids_cudf_ColumnView_applyBooleanMask(
   CATCH_STD(env, 0);
 }
 
+JNIEXPORT jboolean JNICALL Java_ai_rapids_cudf_ColumnView_hasNonEmptyNulls(
+    JNIEnv *env, jclass, jlong column_view_handle) {
```

Comment:
IMO `column_view_handle` is a much more descriptive name and doesn't make sense to change it to a less descriptive name

Reply:
I think this is a minor quibble. My $0.02: it's not a pointer until it has been cast from the opaque `jlong`. I would keep it as `handle`. (Plus, it's consistent with the other methods.)

`input_handle` is descriptive, because it indicates that it's the input. `column_view` is more ambiguous and possibly redundant, because it's already a method in `ColumnView`.

(Again, very minor quibble.)

Reply:
`column_view_handle` seems too long name. In addition, the existing functions already use `input_handle` like https://github.com/rapidsai/cudf/blob/d9bd29a6e7a382564790492b6cebe75e4d1cefc9/java/src/main/native/src/ColumnViewJni.cpp#L683. So please enforce consistency.

Reply:
This feels so minor to me. If the name is wrong or confusing, then yes lets change it. But most anything else is just a nit and really it should be up to @razajafri to decide if he uses other's advice or not.